src/main/java/com/github/nicholasmoser/gnt4/seq/comment/Comments.java

```diff
@@ -25,178 +29,190 @@ public class Comments {
     cho0000Comments.put(0x5EF0, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.CHO_0000, cho0000Comments);
 
-    Map<Integer, String> dog0000Comments = new HashMap<>();
+    Multimap<Integer, String> dog0000Comments = ArrayListMultimap.create();
     dog0000Comments.put(0x5D30, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.DOG_0000, dog0000Comments);
 
-    Map<Integer, String> gai0000Comments = new HashMap<>();
+    Multimap<Integer, String> gai0000Comments = ArrayListMultimap.create();
     gai0000Comments.put(0x5C90, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.GAI_0000, gai0000Comments);
 
-    Map<Integer, String> gar0000Comments = new HashMap<>();
+    Multimap<Integer, String> gar0000Comments = ArrayListMultimap.create();
     gar0000Comments.put(0x5CE0, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.GAR_0000, gar0000Comments);
 
-    Map<Integer, String> hak0000Comments = new HashMap<>();
+    Multimap<Integer, String> hak0000Comments = ArrayListMultimap.create();
     hak0000Comments.put(0x61A0, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.HAK_0000, hak0000Comments);
 
-    Map<Integer, String> hi20000Comments = new HashMap<>();
+    Multimap<Integer, String> hi20000Comments = ArrayListMultimap.create();
     hi20000Comments.put(0x5E40, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.HI2_0000, hi20000Comments);
 
-    Map<Integer, String> hin0000Comments = new HashMap<>();
+    Multimap<Integer, String> hin0000Comments = ArrayListMultimap.create();
     hin0000Comments.put(0x5E10, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.HIN_0000, hin0000Comments);
 
-    Map<Integer, String> ino0000Comments = new HashMap<>();
+    Multimap<Integer, String> ino0000Comments = ArrayListMultimap.create();
     ino0000Comments.put(0x5CF0, "// Scale chr model larger by 1.1x");
     allComments.put(Seqs.INO_0000, ino0000Comments);
 
-    Map<Integer, String> iru0000Comments = new HashMap<>();
+    Multimap<Integer, String> iru0000Comments = ArrayListMultimap.create();
     iru0000Comments.put(0x5CF0, "// Scale chr model larger by 1.1x");
+
+    iru0000Comments.put(0xA1C, "// Many actions end by branch and linking to here");
+    iru0000Comments.put(0xAA8, "// Action Offset Lookup Branch #1");
+
+    iru0000Comments.put(0xB08, "// Action Offset Lookup Branch #2 (Happy path)");
+    iru0000Comments.put(0xB18, "// Load current action id into gpr2");
+    iru0000Comments.put(0xB20, "// Convert action id to table offset");
+    iru0000Comments.put(0xB28, "// Load memory address of chr_tbl into gpr19");
+    iru0000Comments.put(0xB30, "// Loads action offset from chr_tbl");
+
+    iru0000Comments.put(0x9B88, "// Set chr_tbl address to *seq_p_sp->field_0x260 + offset 0x230");
+    iru0000Comments.put(0x9E7C, "// Set chr_tbl address to *seq_p_sp->field_0x260 + offset 0x230");
```

Comment:
These should be *seq_p_sp->field_0x38 + offset 0x230

Reply:
Good catch, fixed in https://github.com/NicholasMoser/GNTool/pull/97/commits/aeff254e1fa5072e38b252ecb7ae6d99946bb0dc